sungrow/sungrow-registers.json

```diff
@@ -0,0 +1,642 @@
+{
+    "className": "Sungrow",
+    "protocol": "BOTH",
```

Comment:
You mentioned you get errors when using only TCP. I tested this with the latest rebase on `fix-solax-sequential-polling`  branch and had no issues. 
Please let me know if you still have problems with using only TCP as protocol.